src/index.ts

```diff
@@ -127,6 +127,15 @@ export class EpicTest<Action extends IAction<any>, State, Dependencies> {
       );
       helpers.expectObservable(output).toBe(marbles, expectations);
     });
+
+    return this;
+  }
+  /**
+   * Executes a given callback function
+   * @param callback - function to execute
+   */
+  public after<T>(callback: () => T | Promise<T>) {
+    callback();
```

Comment:
`return` the T from the callback too so that it can be awaited, as necessary :)

Reply:
Actually, I think I am missing a set of parenthesis. Shouldn't it be something like `callback (() => T) | Promise<T>` ? But on the second thought do we need Promise there at all? What is the use case you are thinking of?

Reply:
I don't think we need the promise there, no. It should just be `public after<T>(callback: () => T): T {`. The use case is that if we want to do some async verification or teardown in after(), we'll want to return the promise from inside the callback so that we can await it.

Reply:
Makes sense, updated